diff --git i/lib/PhpParser/NodeAbstract.php w/lib/PhpParser/NodeAbstract.php
index abd5fd6..297e54e 100644
--- i/lib/PhpParser/NodeAbstract.php
+++ w/lib/PhpParser/NodeAbstract.php
@@ -21,7 +21,7 @@
      * @return string Type of the node
      */
     public function getType() {
-        return strtr(substr(rtrim(get_class($this), '_'), 15), '\\', '_');
+        return strtr(substr(rtrim(get_class($this), '_'), 29), '\\', '_');
     }

     /**
